content/blog/jt15s/2020-12-31_hello_from_the_haiku_promotion_team.md

```diff
@@ -0,0 +1,56 @@
++++
+type = "blog"
+title = "Hello from the Haiku Promotion 'Team'!"
+author = "jt15s"
+date = "2020-12-31 21:33:25+11:00"
+tags = ["haiku", "software"]
++++
+Hi there, I'm jt15s, a new Haiku community member! I've been following the project for a few years now since I stumbled upon an article about Haiku. 
+
+If you read the title (which I'm assuming you did), you’re probably going, "wait, we have a promotion team now?" Well, yes, now we do. As an informal “team” of two (currently), we are super excited to share our plans to help promote Haiku and make more people aware of it.
+
+## Who's in it?
+Me, as well as scott_puopolo18 as of now. We are also working with Haikunauts and BeSly, two Haiku-related resources. However, anyone who wishes to help is more than welcome to! Reach out to me (jt15s) [on the forums](https://discuss.haiku-os.org) if you are interested. If you don't have a forum account yet, create one - a forum account is free! We are also looking for the following informal roles:
```

Comment:
@humdingerb I've fixed those parts and have added my real name. I'm just waiting for Scott's reply now.

Reply:
> Using nicknames is fine too. I like to normalize that, so that people not willing to reveal their real name for whatever reason don't feel singled out.

I suppport that in general; I'm pretty much only known by my nickname.
In case of the PR&marketing team though, I can imagine that their contact to business types and journalists may be easier with the additional establishment of trust by knowing the real name of your counterpart.
Don't feel obliged to, though.